src/Core/hco_types_mod.F90

```diff
@@ -94,9 +94,9 @@ MODULE HCO_TYPES_MOD
      INTEGER                 :: HcoID      ! HEMCO species ID
      INTEGER                 :: ModID      ! Model species ID
      CHARACTER(LEN= 31)      :: SpcName    ! species names
-#ifdef HEMCO_CESM
+#ifdef MODEL_CESM
      INTEGER                 :: DimMax     ! Maximum dimensions supported: 2 (2-D), 3 (3-D)
-                                           ! HEMCO_CESM only, as 3-D emissions must be listed
```

Comment:
If we're scrubbing the model of using "HEMCO_CESM" here's another one:
https://github.com/geoschem/HEMCO/blob/4e818813da35bf76dbcee2da6ecc598b9c89b1a7/src/Core/hcoio_read_std_mod.F90#L1357

Reply:
Fixed, thanks!